test/core/utils/preload-media.js

```diff
@@ -17,26 +14,9 @@ describe('axe.utils.preloadMedia', function() {
 	});
 
 	afterEach(function() {
-		axe.utils.preloadMedia = origFn;
 		fixture.innerHTML = '';
 	});
 
-	it('invokes utils.preloadMedia and passes the treeRoot property', function(done) {
```

Comment:
This test does not test anything. It mocks a function and asserts that the function receives the passed in property